<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BdapAddUserDialog</class>
 <widget class="QDialog" name="BdapAddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add BDAP User</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_userID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelCommonName">
      <property name="text">
       <string>Common name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_commonName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelRegistrationDays">
      <property name="text">
       <string>Registration days:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_registrationDays"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelUserId">
      <property name="text">
       <string>User ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create a new address</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
   <property name="icon">
    <iconset resource="../dynamic.qrc">
     <normaloff>:/icons/drk/add</normaloff>:/icons/drk/add</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create a new address</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="../dynamic.qrc">
     <normaloff>:/icons/drk/transaction_conflicted</normaloff>:/icons/drk/transaction_conflicted</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../dynamic.qrc"/>
 </resources>
 <connections/>
</ui>
